.circleci/config.yml

```diff
@@ -152,6 +152,22 @@ jobs:
             just simulate-council
             just prepare-json
             just simulate-council # simulate again to make sure the json is still valid
+  just_simulate_sep-metal-001-MCP-L1:
+    docker:
+      - image: us-docker.pkg.dev/oplabs-tools-artifacts/images/ci-builder:v0.35.0
```

Comment:
`ci-builder:v0.47.0` has `yq` in it, see https://github.com/ethereum-optimism/optimism/blob/3e4430ee40ab72edb51ed11e5fa277ee7ae01746/ops/docker/ci-builder/Dockerfile#L61

Reply:
Ah nice, now using that per 5290ca0

Reply:
Hm, CI now fails with `just: command not found`, but I do see it being installed [here](https://github.com/ethereum-optimism/optimism/blob/3e4430ee40ab72edb51ed11e5fa277ee7ae01746/ops/docker/ci-builder/Dockerfile#L22)

https://app.circleci.com/pipelines/github/ethereum-optimism/superchain-ops/701/workflows/d27b7549-f6dd-4569-8edb-5b6a1a48fab4/jobs/6036

Reply:
apologies for the thrash, we can revert and figure out later why its not working

Reply:
https://github.com/ethereum-optimism/optimism/pull/10158

Reply:
Reverted this for now and will follow up with a separate PR using the new ci-builder later